symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdkBuilder.java

```diff
@@ -0,0 +1,153 @@
+package com.symphony.bdk.core;
+
+import com.symphony.bdk.core.auth.AuthenticatorFactory;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.client.ApiClientFactory;
+import com.symphony.bdk.core.config.BdkConfigLoader;
+import com.symphony.bdk.core.config.exception.BdkConfigException;
+import com.symphony.bdk.core.config.model.BdkConfig;
+
+import com.symphony.bdk.core.util.ServiceLookup;
+import com.symphony.bdk.http.api.ApiClientBuilderProvider;
+
+import lombok.Generated;
+import org.apiguardian.api.API;
+
+import java.io.InputStream;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Fluent builder for advanced configuration of the {@link SymphonyBdk} entry point.
+ */
+@Generated // a bit tricky to get acceptable coverage
+@API(status = API.Status.STABLE)
```

Comment:
should we mark it as experimental for now? (and even leave it experimental for a while as it opens a lot of things in the BDK) (although I'm not sure the @api annotation is really visible for users)

Reply:
At least visible in Javadoc: 
![Screen Shot 2021-02-17 at 10 08 04](https://user-images.githubusercontent.com/39826516/108181336-102d1880-7108-11eb-9766-27dbd1dbec37.png)